modules/installation-osp-deleting-bootstrap-resources.adoc

```diff
@@ -4,30 +4,22 @@
 // * installing/installing_openstack/installing-openstack-kuryr.adoc
```

Comment:
Should this assembly be `installing-openstack-user-kuryr.adoc`? Just noticed it when I tried getting to the preview for this module.

Reply:
Yeah, whoops.